### ISSUE TITLE
builder: plan9/386 builder is failing since merge of dev.debug

### ISSUE THREAD

**Issue by 0intro** [golang/go#21406]
Since the merge of origin/dev.debug branch into master (commit [385cd6681b](https://github.com/golang/go/commit/385cd6681bd5dcb1dd2678d1d8db93b4f3d751f9)), the plan9/386 builder is failing to build Go.

```
# bootstrap/cmd/compile/internal/gc
run compiler with -v for register allocation sites
/tmp/workdir/go/src/cmd/compile/internal/gc/pgen.go:628[/tmp/workdir/go/pkg/bootstrap/src/bootstrap/cmd/compile/internal/gc/pgen.go:631]: internal compiler error: out of fixed registers

goroutine 1 [running]:
runtime/debug.Stack(0x0, 0x0, 0x0)
	/home/bradfitz/go-plan9-386-bootstrap/src/runtime/debug/stack.go:24 +0x7f
cmd/compile/internal/gc.Fatalf(0x4b1568, 0x16, 0x0, 0x0, 0x0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/subr.go:158 +0x23a
cmd/compile/internal/gc.Regalloc(0x1527d140, 0x107a5980, 0x1527d080)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gsubr.go:722 +0x39c
cmd/compile/internal/x86.ginscmp(0x3c, 0x107a5980, 0x1527cf60, 0x1527d080, 0xffffffff, 0x10)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/x86/gsubr.go:657 +0x2cf
cmd/compile/internal/gc.cgen_dottype(0x11b519e0, 0x1527c5a0, 0x0, 0x0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:408 +0x16a
cmd/compile/internal/gc.cgen_wb(0x11b519e0, 0x1527c5a0, 0x30794000)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:60 +0x1ea
cmd/compile/internal/gc.Cgen(0x11b519e0, 0x1527c5a0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:18 +0x2d
cmd/compile/internal/gc.Cgenr(0x11b519e0, 0x1527c5a0, 0x1527c4e0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:954 +0x1c8
cmd/compile/internal/gc.Igen(0x11b51a40, 0x1527c5a0, 0x1527c4e0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:1659 +0x2f3
cmd/compile/internal/gc.cgen_wb(0x11b51a40, 0x1527c4e0, 0x1527c400)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:523 +0x184d
cmd/compile/internal/gc.Cgen(0x11b51a40, 0x1527c4e0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:18 +0x2d
cmd/compile/internal/x86.cgen64(0x11b51aa0, 0x1527b380)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/x86/cgen64.go:76 +0x313
cmd/compile/internal/gc.cgen_wb(0x11b51aa0, 0x1527b380, 0x0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:327 +0x154b
cmd/compile/internal/gc.Cgen(0x11b51aa0, 0x1527b380)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:18 +0x2d
cmd/compile/internal/gc.Mgen(0x11b51aa0, 0x1527b380, 0x1527b320)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:906 +0xd8
cmd/compile/internal/gc.cgen_wb(0x11b51b00, 0x1527b320, 0x1527b300)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:459 +0x2786
cmd/compile/internal/gc.Cgen(0x11b51b00, 0x1527b320)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:18 +0x2d
cmd/compile/internal/gc.cgen_wb(0x11b51b00, 0x11b515c0, 0x0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/cgen.go:101 +0x510
cmd/compile/internal/gc.Cgen_as_wb(0x11b515c0, 0x11b51b00, 0x5100)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:936 +0x137
cmd/compile/internal/gc.Cgen_as(0x11b515c0, 0x11b51b00)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:887 +0x2d
cmd/compile/internal/gc.gen(0x11b51b60)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:834 +0x1bd
cmd/compile/internal/gc.Genlist(0x151cb6e0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:212 +0x6c
cmd/compile/internal/gc.gen(0x11b51440)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:783 +0xc9a
cmd/compile/internal/gc.Genlist(0x152268f0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:212 +0x6c
cmd/compile/internal/gc.gen(0x11b4d860)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:768 +0xa88
cmd/compile/internal/gc.Genlist(0x151cba20)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:212 +0x6c
cmd/compile/internal/gc.gen(0x11b4b860)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:768 +0xa88
cmd/compile/internal/gc.Genlist(0x151cba30)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/gen.go:212 +0x6c
cmd/compile/internal/gc.genlegacy(0x1087a184, 0x15234b40, 0x15234bc0)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/pgen.go:496 +0x55
cmd/compile/internal/gc.compile(0x11b44780)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/pgen.go:489 +0x1016
cmd/compile/internal/gc.funccompile(0x11b44780)
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/dcl.go:1283 +0x19b
cmd/compile/internal/gc.Main()
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/gc/main.go:474 +0x1ecd
cmd/compile/internal/x86.Main()
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/internal/x86/galign.go:85 +0x2fb
main.main()
	/home/bradfitz/go-plan9-386-bootstrap/src/cmd/compile/main.go:30 +0x116
```

See https://build.golang.org/log/d19214ad27f6c1dace1c3f4674b524fda9842f7e

**Comment by 0intro** [comment]
@bradfitz, would it be possible to update the bootstrap package on the plan9/386 builder on GCE to Go 1.8.3?

A binary package is available here:

http://9legacy.org/download/go/go1.8.3-plan9-386-bootstrap.tbz

Thanks.

**Comment by 0intro** [comment]
CC @adams-sarah, @cybrcodr and @jessfraz, who are in charge of the builders, since @bradfitz is on vacation.

**Comment by adams-sarah** [comment]
cc @aclements

**Comment by aclements** [comment]
What bootstrap compiler are we running on the other builders? We shouldn't depend on something newer than 1.4.x (unless the policy is different for plan9?)

**Comment by 0intro** [comment]
If I recall correctly, the plan9/386 builder is running Go 1.6. The policy is different for Plan 9, because the Go 1.4 runtime wasn't mature enough. The other Plan 9 builders (amd64 and arm) are currently running Go 1.8.3 (from the same binary package).

**Comment by aclements** [comment]
Okay. So do we basically treat plan9 like we treat GOARCHes that weren't supported by 1.4 (e.g., you have to cross-compile to bootstrap)?

**Comment by 0intro** [comment]
Yes.

**Comment by aclements** [comment]
Thanks. In that case, I agree that we should upgrade the plan9/386 builder's bootstrap to 1.8.3 to match the other plan9 builders.

**Comment by 0intro** [comment]
Could you build Go 1.8.3 with GOOS=plan9 and GOARCH=386 (or use the binary package I provide) and upload it to https://storage.googleapis.com/$BUCKET/gobootstrap-plan9-386.tar.gz?

So the future plan9/386 builds will use the new bootstrap, which should get rid of the current build failure.

**Comment by 0intro** [comment]
I've checked the Go version currently used to bootstrap Go on the plan9/386 builder ([gobootstrap-plan9-386.tar.gz](https://storage.googleapis.com/go-builder-data/gobootstrap-plan9-386.tar.gz)).

```
% hget -o /tmp/gobootstrap-plan9-386.tar.gz https://storage.googleapis.com/go-builder-data/gobootstrap-plan9-386.tar.gz
% mkdir /tmp/go
% cd /tmp/go
% tar xf /tmp/gobootstrap-plan9-386.tar.gz
% bin/go version
go version devel +b64f549 Fri Apr 1 02:08:56 2016 +0000 plan9/386
```

This is commit [b64f549](https://github.com/golang/go/commit/b64f549ba95fb9115afb1db8ae594b9442c45a6e) from 2016-03-30. This is located during the development phase between Go 1.6 and Go 1.7.

I can reproduce the same issue when bootstraping the [current](https://github.com/golang/go/commit/23cd87eb0a2d49a3208824feaf34d8b852da422f) Go version with Go version [b64f549](https://github.com/golang/go/commit/b64f549ba95fb9115afb1db8ae594b9442c45a6e) on linux/386.

Build Go [b64f549](https://github.com/golang/go/commit/b64f549ba95fb9115afb1db8ae594b9442c45a6e):

```
$ git clone https://go.googlesource.com/go /tmp/go-bootstrap
$ cd /tmp/go-bootstrap
$ git checkout b64f549
$ cd src
$ GOHOSTARCH=386 GOARCH=386 ./make.bash
```

Build [current](https://github.com/golang/go/commit/23cd87eb0a2d49a3208824feaf34d8b852da422f) from Go [b64f549](https://github.com/golang/go/commit/b64f549ba95fb9115afb1db8ae594b9442c45a6e) bootstrap on linux/386:

```
$ git clone https://go.googlesource.com/go /tmp/go
$ cd /tmp/go/src
$ GOROOT_BOOTSTRAP=/tmp/go-bootstrap GOHOSTARCH=386 GOARCH=386 ./make.bash
[...]
bootstrap/cmd/compile/internal/gc
# bootstrap/cmd/compile/internal/gc
run compiler with -v for register allocation sites
/tmp/go/src/cmd/compile/internal/gc/pgen.go:628[/tmp/go/pkg/bootstrap/src/bootstrap/cmd/compile/internal/gc/pgen.go:631]: internal compiler error: out of fixed registers

goroutine 1 [running]:
runtime/debug.Stack(0x0, 0x0, 0x0)
        /tmp/go-bootstrap/src/runtime/debug/stack.go:24 +0x80
cmd/compile/internal/gc.Fatalf(0x84fd90b, 0x16, 0x0, 0x0, 0x0)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/subr.go:158 +0x23b
cmd/compile/internal/gc.Regalloc(0x1d14c1e0, 0x189d1480, 0x1d14c120)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gsubr.go:722 +0x39d
cmd/compile/internal/x86.ginscmp(0x3c, 0x189d1480, 0x1d14c000, 0x1d14c120, 0xffffffff, 0x10)
        /tmp/go-bootstrap/src/cmd/compile/internal/x86/gsubr.go:657 +0x2d0
cmd/compile/internal/gc.cgen_dottype(0x19c80f00, 0x1d14b620, 0x0, 0x0)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:408 +0x16b
cmd/compile/internal/gc.cgen_wb(0x19c80f00, 0x1d14b620, 0xf76d9200)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:60 +0x1eb
cmd/compile/internal/gc.Cgen(0x19c80f00, 0x1d14b620)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:18 +0x2e
cmd/compile/internal/gc.Cgenr(0x19c80f00, 0x1d14b620, 0x1d14b560)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:954 +0x1c9
cmd/compile/internal/gc.Igen(0x19c80f60, 0x1d14b620, 0x1d14b560)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:1659 +0x2f4
cmd/compile/internal/gc.cgen_wb(0x19c80f60, 0x1d14b560, 0x1d14b500)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:523 +0x182a
cmd/compile/internal/gc.Cgen(0x19c80f60, 0x1d14b560)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:18 +0x2e
cmd/compile/internal/x86.cgen64(0x19c80fc0, 0x1d14a420)
        /tmp/go-bootstrap/src/cmd/compile/internal/x86/cgen64.go:76 +0x314
cmd/compile/internal/gc.cgen_wb(0x19c80fc0, 0x1d14a420, 0x0)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:327 +0x1534
cmd/compile/internal/gc.Cgen(0x19c80fc0, 0x1d14a420)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:18 +0x2e
cmd/compile/internal/gc.Mgen(0x19c80fc0, 0x1d14a420, 0x1d14a3c0)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:906 +0xd9
cmd/compile/internal/gc.cgen_wb(0x19c81020, 0x1d14a3c0, 0x1d14a300)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:459 +0x2769
cmd/compile/internal/gc.Cgen(0x19c81020, 0x1d14a3c0)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:18 +0x2e
cmd/compile/internal/gc.cgen_wb(0x19c81020, 0x19c80ae0, 0x0)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/cgen.go:101 +0x511
cmd/compile/internal/gc.Cgen_as_wb(0x19c80ae0, 0x19c81020, 0x5100)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:936 +0x138
cmd/compile/internal/gc.Cgen_as(0x19c80ae0, 0x19c81020)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:887 +0x2e
cmd/compile/internal/gc.gen(0x19c81080)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:834 +0x1be
cmd/compile/internal/gc.Genlist(0x1c60de20)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:212 +0x6d
cmd/compile/internal/gc.gen(0x19c80960)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:783 +0xc9b
cmd/compile/internal/gc.Genlist(0x1c679030)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:212 +0x6d
cmd/compile/internal/gc.gen(0x19c7ed80)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:768 +0xa89
cmd/compile/internal/gc.Genlist(0x1c678160)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:212 +0x6d
cmd/compile/internal/gc.gen(0x19c7ad80)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:768 +0xa89
cmd/compile/internal/gc.Genlist(0x1c678170)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/gen.go:212 +0x6d
cmd/compile/internal/gc.genlegacy(0x189de184, 0x1c689140, 0x1c6891c0)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/pgen.go:496 +0x56
cmd/compile/internal/gc.compile(0x19c71c80)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/pgen.go:489 +0x1014
cmd/compile/internal/gc.funccompile(0x19c71c80)
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/dcl.go:1283 +0x19c
cmd/compile/internal/gc.Main()
        /tmp/go-bootstrap/src/cmd/compile/internal/gc/main.go:474 +0x1ece
cmd/compile/internal/x86.Main()
        /tmp/go-bootstrap/src/cmd/compile/internal/x86/galign.go:85 +0x2fc
main.main()
        /tmp/go-bootstrap/src/cmd/compile/main.go:30 +0x117

go tool dist: FAILED: /tmp/go-bootstrap/bin/go install -gcflags=-l -tags=math_big_pure_go -v bootstrap/cmd/...: exit status 2
```

So, I suspect that at some point during the Go 1.7 development phase, there was an issue when gc could run out of registers on 386 architecture, which was later fixed before Go 1.7 release and this issue was exposed yesterday when building [b64f549](https://github.com/golang/go/commit/385cd6681bd5dcb1dd2678d1d8db93b4f3d751f9) with this version of gc.

Consequently, bootstrapping from Go 1.8.3 should fix this issue on the plan9/386 builder.

**Comment by aclements** [comment]
It looks like I don't have permission on the go-builder-data bucket (or I have no idea what I'm doing, which is also possible).

@adams-sarah, @cybrcodr or @jessfraz, could you update the plan9/386 bootstrap Go to 1.8.3?

**Comment by cybrcodr** [comment]
@aclements, I can give it a shot, seems like I'm at least able to run ...
$ gsutil acl get gs://go-builder-data/
w/o any permission issues.

Where do I get the bootstrap copy from?  Or how do I produce one?

I probably should back-up the existing one before replacing it too, just in case:)

**Comment by adams-sarah** [comment]
@cybrcodr gobootstrap is just a GOROOT tarball, just like the ones you find on https://golang.org/dl/
just make sure the new one is at the same gcs path w/ same name and you shouldn't have to do anything else :)

for intermediate backup you can just dl it. 

thx for taking this!

**Comment by cybrcodr** [comment]
I don't think we have a pre-built tarball for plan9/386 though.  I guess I can cross compile to produce one.  I'll use the go1.8.3.src.tar.gz as source.

I did a backup for the current one --
$ gsutil ls gs://go-builder-data/gobootstrap-plan9-386*
gs://go-builder-data/gobootstrap-plan9-386-20160401.tar.gz
gs://go-builder-data/gobootstrap-plan9-386.tar.gz

**Comment by bradfitz** [comment]
Careful. They're not identical to the /dl/ binaries.

Two notable differences:

1) the bootstrap ones do not have the `go/` directory prefix on each file:

```
bradfitz@gdev:~/src/golang.org/x/sys/unix$ curl --silent https://storage.googleapis.com/go-builder-data/gobootstrap-plan9-386.tar.gz | tar -zt | head -20
AUTHORS
bin/
bin/go
CONTRIBUTING.md
CONTRIBUTORS
favicon.ico
lib/
lib/time/
lib/time/README
lib/time/zoneinfo.zip
lib/time/update.bash
LICENSE
misc/
misc/sortac/
misc/sortac/sortac.go
misc/android/
misc/android/README
misc/android/cleaner.go
misc/android/go_android_exec.go
misc/linkcheck/
bradfitz@gdev:~/src/golang.org/x/sys/unix$ curl --silent https://storage.googleapis.com/golang/go1.8.3.darwin-amd64.tar.gz | tar -zt | head -20
go/
go/AUTHORS
go/CONTRIBUTING.md
go/CONTRIBUTORS
go/LICENSE
go/PATENTS
go/README.md
go/VERSION
go/api/
go/api/README
go/api/except.txt
go/api/go1.1.txt
go/api/go1.2.txt
go/api/go1.3.txt
go/api/go1.4.txt
go/api/go1.5.txt
go/api/go1.6.txt
go/api/go1.7.txt
go/api/go1.8.txt
go/api/go1.txt
```

2) the bootstrap versions are stripped with all the useless parts removed. There's a bug somewhere about bootstrap tarballs and shrinking them that says the command(s) to run to delete extra dirs & files/binaries that are unneeded during bootstrap. This step is optional but speeds up builds.

**Comment by bradfitz** [comment]
The trimming notes are at https://github.com/golang/go/issues/9797#issuecomment-204592820

I generally upload to GCS with golang.org/x/build/cmd/upload (use `--public` to set public ACLs)

**Comment by bradfitz** [comment]
(And somebody will inevitably suggest this can be automated, which is true, but do the https://xkcd.com/1205/ math)

Currently bootstrap.bash does the bulk of it. Then you just need to recreate the tar.gz created by bootstrap bash because it creates a tar.bz2 which is not supported by the buildlet.

**Comment by cybrcodr** [comment]
Ok.  It took me a while to figure this out.  I initially ran "GOOS=plan9 GOARCH=386 make.bash", but realized that did not produced a plan9 go binary.  Since you mentioned about bootstrap.bash, I had initially thought that was only for packaging, but after looking at the script, I realized that is THE script to build a bootstrap.

I also noticed in the trimming notes you mentioned above about genbootstrap, so I tried using that on my Mac, but somehow the produced tarball had strange tar entries ...

```
-rw-r--r--  0 herbie eng       226 Aug 14 18:13 ./._AUTHORS
-rw-r--r--  0 herbie eng     33243 Aug 14 18:13 ./AUTHORS
drwxrwxr-x  0 herbie eng         0 Aug 14 18:15 ./bin/
-rw-r--r--  0 herbie eng       226 Aug 14 18:13 ./._CONTRIBUTING.md
-rw-r--r--  0 herbie eng      1366 Aug 14 18:13 ./CONTRIBUTING.md
-rw-r--r--  0 herbie eng       226 Aug 14 18:13 ./._CONTRIBUTORS
-rw-r--r--  0 herbie eng     45710 Aug 14 18:13 ./CONTRIBUTORS
-rw-r--r--  0 herbie eng       226 Aug 14 18:13 ./._favicon.ico
-rw-r--r--  0 herbie eng      5686 Aug 14 18:13 ./favicon.ico
-rwxr-xr-x  0 herbie eng       226 Aug 14 18:13 ./._lib
drwxr-xr-x  0 herbie eng         0 Aug 14 18:13 ./lib/
```

I ran the build again on a Linux box and it looks cleaner.  I then attempted to use the upload tool as well, and I had to generate a OAuth2 json token for that.  Somehow I didn't need this token file when using gsutil to make the backup copy.

Anyways, it's been updated now.  Hope it works!

And thanks for the tips bradfitz@!

**Comment by cybrcodr** [comment]
Learned something new.  How far back did you retry it for?

The latest failures showing up on build.golang.org have the following similar failed tests ...
```
ok  	cmd/link	0.034s
--- FAIL: TestSizes (0.58s)
	dwarf_test.go:217: error reading DWARF: no DWARF data in Plan 9 file
FAIL
FAIL	cmd/link/internal/ld	0.615s
2017/08/14 22:56:07 Failed: exit status: 'go 5534: 1'
```

Particular ones that I looked at are ...
Commit: 9223adcc2c48f4b7f60ff10001bb35f028e6126a
Time: 15 Aug 03:07 GMT
Log: https://build.golang.org/log/e0876e99efe93211f40f2d7026c2dee4c39c0a7f

Commit: fcf445dce22f96f874932b37962acba40d51e000
Time: 15 Aug 01:52 GMT
Log: https://build.golang.org/log/4f0af16c519b685c7babf7610d1beb975651c749

and a couple of other ones around those commits.

I wonder if this has to do with bootstrap I uploaded.
